tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -295,6 +295,7 @@ cc_library(
         "@llvm//:object",
         "@llvm//:support",
         "@llvm//:target",
+        "@llvm//:polly",
```

Comment:
Thanks for reviewing :)

How can I add a conditional in the cc_library() rule?

I could add a flag to check if the user wants to add Polly's passes in the `tensorflow/compiler/xla/service/cpu/compiler_functor.cc` file.